include/gsl/span

```diff
@@ -262,22 +289,22 @@ namespace details
         // algorithm calls
         friend constexpr void _Verify_range(span_iterator lhs, span_iterator rhs) noexcept
         { // test that [lhs, rhs) forms a valid range inside an STL algorithm
-            Expects(lhs.span_ == rhs.span_        // range spans have to match
-                    && lhs.index_ <= rhs.index_); // range must not be transposed
+            Expects(lhs.begin_ == rhs.begin_    // range spans have to match
+                    && lhs.end_ <= rhs.end_); // range must not be transposed
         }
 
         constexpr void _Verify_offset(const difference_type n) const noexcept
-        { // test that the iterator *this + n is a valid range in an STL
+        { // test that *this + n is within the span of this iterator STL
             // algorithm call
```

Comment:
"test that *this + n is within the span of this iterator" makes grammatical sense (although using "span" in any sense other than "gsl::span" is somewhat confusing). However, "STL algorithm call" is a relic of the previous comment.